### PR TITLE
🌐 Add translation comments and make more strings translatable

### PR DESCRIPTION
**Pull request by daPhipz**
Wow, this is quite a big one xD

I checked `flypie.pot` for strings that seemed unclear to me and have the potential to be misunderstood when translating them.
Furthermore, I made the strings in the `easing-mode-list` translatable. No hard feelings if this was never intended ;)

I will comment on some of the changes I made, to be more clear about my intentions.

### CODE REVIEW

File: settings/settings.ui [hunk 1]
```diff
@@ -1094,7 +1094,7 @@ Once you feel comfortable, click &lt;b&gt;next&lt;/b&gt; to learn more details!<
                             <property name="can_focus">False</property>
                             <property name="margin_top">10</property>
                             <property name="margin_bottom">10</property>
-                            <property name="label" translatable="yes" comments="The text of the big tutorial button. Using two lines looks quite good.">&lt;big&gt;&lt;b&gt;Open the example menu!&lt;/b&gt;&lt;/big&gt;</property>
+                            <property name="label" translatable="yes" comments="The text of the big tutorial button.">&lt;big&gt;&lt;b&gt;Open the example menu!&lt;/b&gt;&lt;/big&gt;</property>
```
Comment (daPhipz) [inline settings/settings.ui:1097]: I think there was some design change, wasn't there? One single line does look good, too IMO

Comment (Schneegans) [reply settings/settings.ui:1097]: Yeah, I just forgot to update the comment.

**Comment by Schneegans** [comment]
Wow. Now I am honestly impressed. Thank you for this effort! I'll run clang-format after the merge.

**Comment by daPhipz** [comment]
Thank you for the kind words :blush:

**Pull request merged**